apps/api/src/persistence/database/utilities/convertArangoDocumentHandleToCompositeIdentifier.ts

```diff
@@ -0,0 +1,37 @@
+import { EntityId, isResourceId } from '../../../domain/types/ResourceId';
+import { InternalError } from '../../../lib/errors/InternalError';
+import { ArangoCollectionID, isArangoCollectionID } from '../types/ArangoCollectionId';
+import { ArangoDocumentHandle } from '../types/ArangoDocumentHandle';
+
+type ArangoCompositeIdentifier = {
+    collection: ArangoCollectionID;
+    id: EntityId;
+};
+
+export default (handle: ArangoDocumentHandle): ArangoCompositeIdentifier => {
```

Comment:
This is a step between getting a domain composite identifier?

Reply:
It could be, although that is usually not necessary. The repository already knows the `resourceOrNoteType` it is for, so only the `id` (Arango `_key`) needs to pass through the db layer to the repository.

This is being used to parse the _to \ _from for a `category_edges` document. 

However, you bring up a good point. We may want to convert `members` of tags and categories to the database representation so that `type` becomes instead `collection`.